### PR TITLE
separated main.go and created primaryHandler.go

### PR DESCRIPTION
**Pull request by mtrinh11**

### CODE REVIEW

**Comment by mtrinh11** [comment]
fixes https://github.com/xmidt-org/tr1d1um/issues/237

File: main.go [hunk 1]
```diff
@@ -18,26 +18,19 @@
 package main
```
Comment (kristinapathak) [inline main.go:18]: We should fix this copyright to say 2021 instead of 2017.

File: primaryHandler.go [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+
+type CapabilityConfig struct {
+	Type            string
+	Prefix          string
+	AcceptAllMethod string
+	EndpointBuckets []string
+}
+
+// JWTValidator provides a convenient way to define jwt validator through config files
+type JWTValidator struct {
```
Comment (kristinapathak) [inline primaryHandler.go:72]: All structs at the top of the file looks good. 👍

**Review by kristinapathak — APPROVED** [review]
Looks good, one small copyright change.

**Pull request merged**